roles/arteria-sequencing-report-ws/templates/seqreport_app.config.j2

```diff
@@ -17,7 +17,9 @@ nextflow_config:
     nf_profile: {{ seqreports_nf_profile }}
     environment:
         NXF_TEMP: {{ nextflow_env.NXF_TEMP }}
-        NXF_WORK: {{ nextflow_env.NXF_WORK }}
+        NXF_WORK: ${DEFAULT:runfolder_path}/seqreports/nextflow
```

Comment:
Good question, this is actually specific to the `configparser` module, it will substitute this with the content of the `runfolder_path` variable, see here for context: https://github.com/arteria-project/sequencing-report-service/pull/5/commits/aa153e07139c69c9c42d02a23f8a3d404f828de1, and here for more details: https://docs.python.org/3/library/configparser.html#configparser.ExtendedInterpolation

Reply:
Ah, so not shell script at all. Thanks! :)

Reply:
Yeah I also got really confused the first time I encountered this :grimacing: